export_presets.cfg

```diff
@@ -176,7 +176,7 @@ custom_features=""
 export_filter="all_resources"
 include_filter=""
 exclude_filter=""
-export_path="build/index.html"
+export_path="Build/0.14/index.html"
```

Comment:
We should be consistent with naming conventions; i.e., pick between camelCase, PascalCase, snake_case, etc.